examples/plugin/Hello_world/HelloWorld.hh

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (C) 2018 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+
+#ifndef SYSTEM_PLUGIN_HELLOWORLD_HH_
+#define SYSTEM_PLUGIN_HELLOWORLD_HH_
+
+
+#include <ignition/gazebo/System.hh>
+
+namespace hello_world
+{
+
+  class HelloWorld: 
+
+    public ignition::gazebo::System,
+
+    public ignition::gazebo::ISystemPostUpdate
+  
```

Comment:
Mind removing these empty lines from 25 to here?

Reply:
Done